modules/samples-operator-overview.adoc

```diff
@@ -57,9 +57,40 @@ Certain circumstances will result in the Samples Operator bootstrapping itself a
 
 * If the Samples Operator cannot reach link:https://registry.redhat.io[registry.redhat.io] after three minutes on initial start-up after a clean installation.
 * If the Samples Operator detects it is on an IPv6 network.
-* If the Samples Operator detects it is installed on ppc64le and S390x.
+
+However, if Samples Operator also detects an OpenShift Global Proxy is configured, it bypasses those checks.
 
 [IMPORTANT]
 ====
 IPv6 installations are not currently supported by link:https://registry.redhat.io[registry.redhat.io]. The Samples Operator pulls most of the sample imagestreams and images from link:https://registry.redhat.io[registry.redhat.io].
 ====
+
+*Restricted network installation*
+
+Boostrapping as `Removed` when unable to access `registry.redhat.io`, among other things, facilitates restricted network installations when the network restriction is already in place. Not installing the samples immediately when network access is restricted allows the cluster administrator more time to decide if samples are desired before submitting alerts that sample imagestream imports are failing, which it will start doing two hours after initial installation.
+
+*Restricted network installation with initial network access*
+
+Conversely, if a cluster that is intended to be restricted network or disconnected cluster is first installed while network access exists, the Samples Operator will install the content from `registry.redhat.io` since it can access it. If you want the samples operator to still bootstrap as `Removed` in order to defer samples installation until you have decided which samples are desired, set up image mirrors, and so on, then follow the instructions at https://docs.openshift.com/container-platform/4.5/installing/install_config/installing-customizing.html [ED NOTE, fix
```

Comment:
For this link, I think I can add it to the assembly following this section, but it won't be in-line. We can xref between assemblies.

Reply:
ok

Reply:
I put in an additional resources section following this module in the assembly. It's clunky and is probably a temporary solution. If this is a common enough occurrence, I think we'd be better off making it a separate module with a complete procedure. 

I think the content in Using the Samples Operator with an alternate registry (https://docs.openshift.com/container-platform/4.5/openshift_images/samples-operator-alt-registry.html) is also related, so I included a link to that as well.